src/config.ts

```diff
@@ -6,6 +6,8 @@ export const config = {
   maxCompareProducts: 4,
   algoliaAppId: '6MAFUOD23U',
   algoliaApiKey: '7d829396bbf202b30bfe7ad52120bbac',
+  algoliaPlacesAppId: 'plEPUZAA2D2L',
+  algoliaPlacesApiKey: '4c9f0832a65f800e31b0d50f44670b1f',
   algoliaIndexName: 'product',
   compareKeys: [],
   shopkitSrc: 'https://btn.moltin.com',
```

Comment:
We'll need to remove this config (shopkitSrc) as it's not needed anymore with the native checkout.

Reply:
@shaunmaharaj, I still using AlgoliaPlaces to autocomplete address search. Isn't it necessary to use Algolia Places here?
https://github.com/elasticpath/epcc-react-pwa-reference-storefront/blob/161b842839ef64b6228d455268bd018ca2bb81a5/src/PlacesSuggest.tsx#L14

Reply:
@petro97 Algolia configs are correct. We need to remove "shopkitSrc" since it's no longer used anymore in our code 😃

Reply:
Oh sorry, missed it. I'll remove shopkitSrc